README.md

```diff
@@ -74,22 +74,52 @@ These statements must be run through the CipherStash Proxy in order to **decrypt
 **Example:**
 
 ```rb
-Users.findAll(&:encrypted_email)
```

Comment:
@CDThomas I was trying to show a basic SELECT statement here withour any WHERE clauses and highlighting that responses are simply decrypted. I like what you've done here but maybe for the `cs_match_v1` example?

Reply:
That makes sense @calvinbrewer. I reverted the changes in this section (and already updated the match example).